docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -16,7 +16,47 @@ There you can find links to upgrade notes for other versions too.
 - check whether you extended class or method `ImageFacade::copyImages` or used it in your project and make sure it works like you intended ([#851](https://github.com/shopsys/shopsys/pull/851))
     - *(low priority)* remove `/var/www/html/var/cache` folder from your `@main_filesystem` filesystem storage if exists, set as local filesystem storage in path `%kernel.project_dir%` by default
 - use `\Shopsys\FrameworkBundle\Component\Money\Money` class for representing monetary values in the whole application ([#821](https://github.com/shopsys/shopsys/pull/821))
-    - detailed upgrade instruction will be added shortly, see the PR for more information
+    - if you have created your custom entities that store monetary value, change your `decimal` ORM type to `money` type
+    ```diff
+    - * @var string
+    - * @ORM\Column(type="decimal", precision=20, scale=6)
+    + * @var \Shopsys\FrameworkBundle\Component\Money\Money
+    + * @ORM\Column(type="money", precision=20, scale=6)
+    ```
+        - update your custom data objects annotation type to `\Shopsys\FrameworkBundle\Component\Money\Money`
+    - if you are creating value for these custom entities or data objects directly, you have to use `Money::create` method that accepts string or integer or `Money::createFromFloat` method that accepts float and has second mandatory parameter scale
```

Comment:
> if you are creating value for these custom entities

I don't understand what is a "value", please rephrase this sentence

Reply:
please explain what is the "scale", maybe by the link to the documentation

Reply:
I have updated it